packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -65,6 +69,25 @@ const DomainPickerSuggestionItem: FunctionComponent< Props > = ( {
 
 	const labelId = uuid();
 
+	const freeDomainLabel =
+		type === ITEM_TYPE_INDIVIDUAL_ITEM
+			? __( 'Default', __i18n_text_domain__ )
+			: __( 'Free', __i18n_text_domain__ );
+
+	const firstYearIncludedInPaid = isMobile
+		? __( 'Included in paid plans', __i18n_text_domain__ )
+		: createInterpolateElement(
+				__( '<strong>First year included</strong> in paid plans', __i18n_text_domain__ ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'First year included in paid plans' )` *ES Score: 6*